src/Frontend/security/Security.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * This file is part of the sententiaregum application.
+ *
+ * Sententiaregum is a social network based on Symfony2 and ReactJS
+ *
+ * @copyright (c) 2015 Sententiaregum
+ * Please check out the license file in the document root of this application
+ */
+
+/**
+ * @author Benjamin Bieler <benjaminbieler2014@gmail.com>
+ */
+class security {
```

Comment:
hmm, I would refactor the whole security system using reflux:
we need a UserStore that is able to get the securityexpression from the menuconfig in order to evaluate it. After that an event will be triggered using the reflux dispatcher (we DON'T return true because this won't work when using ajax requests in order to check roles for example).
The event should be a class from the Frontend/event directory and should contain information about the user and whether he's allowed to see the menu item.
In your case you just need to build the basic class and trigger __always__ an event that tells the menu that the user is allowed to see every menuitem